__tests__/components/pages/__snapshots__/Landing_test.js.snap

```diff
@@ -49,7 +49,6 @@ exports[`Landing Page Landing Page does render 1`] = `
           "width": 7,
         },
         "shadowOpacity": 0.1,
-        "textAlign": "center",
         "width": 150,
```

Comment:
Interesting that React Native doesn't like 'textAlign'. I'll do some research on an alternative when we're styling things out more.